fix: update android sdk path on windows

ajitzero opened the pull request:
In newer versions of Android Studio on Windows, this Path has been moved out of the Appearance section and into the Languages & Frameworks section.

Screenshot for reference:

<img width="987" height="735" alt="image" src="https://github.com/user-attachments/assets/f015cee3-fbdb-49a7-87ba-2473728bab01" />

---

Taken from:
```text
Android Studio Narwhal | 2025.1.1 Patch 1
Build #AI-251.25410.109.2511.13752376, built on July 9, 2025
Runtime version: 21.0.6+-13391695-b895.109 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 11.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 2048M
Cores: 16
Registry:
  ide.experimental.ui=true
  com.android.studio.ml.activeModel=com.android.studio.ml.AidaModel
```

atanasovg commented:
ping @ikoevska for a review and approval

ErjanGavalji commented on transpilers.md:
```diff
@@ -0,0 +1,42 @@
+
+# Using transpilers in NativeScript
+
+Transpilers allow you to develop your application in languages other than JavaScript, CSS and XML. NativeScript applications can be developed more comfortably when transpilers are used in the project. NativeScript projects support two transpilers out-of-the-box - TypeScript and Babel, although any other transpiler can be integrated through the necessary hooks.
```
I'd use "respective hooks" instead of "necessary hooks"...

ErjanGavalji commented:
:+1:

teobugslayer commented:
I've talked to @ikoevska in person and decided to include this PR as is for 1.5. After that; we shall revisit this article and fix the issues we find.

ajitzero committed:
fix: update android sdk path on windows

In newer versions of Android Studio on Windows, this Path has been moved out of the Appearance section and into the Languages & Frameworks section.

NathanWalker reviewed (approved):
💯
